components/Grid/grid.css

```diff
@@ -0,0 +1,25 @@
+:root {
+    --grid-item-bg-color: rgb(212, 243, 212);
+}
```

Comment:
Hex Values are Better,

Reply:
I used RGB values everywhere. Thanks for the suggestions though.